DMTN-222.tex

```diff
@@ -454,7 +454,302 @@ \section{Batch Process Submission (BPS) Templates}
 \section{Full Calibration Example}
 \label{sec:example}
 
-TODO: DM-48520
+This section contains the commands used to generate, certify, chain, and verify calibrations for LSSTComCam DP1 processing.
+This is adapted from the script attached to DM-48520.
+
+Note that this is not a complete construction of all calibrations; for this we used pre-existing linearity, ptc, and bfk.
+
+In addition to the calibration building, this includes some checkpoints to download mosaic images for visual spot-checking to ensure that nothing has gone terribly wrong before continuing on all the way to verification reports.
+
+\begin{verbatim}
+
+export USER_CALIB_PREFIX=""
+export INSTRUMENT=LSSTComCam
+export TICKET=DM-48520
+export REPO=/repo/main
+export RAW_COLLECTION=LSSTComCam/raw/all
+export CALIB_COLLECTIONS=LSSTComCam/calib/DM-48650,LSSTComCam/calib/DM-47447/gainFixup/ptc.20241107a,LSSTComCam/calib/DM-46360/isrTaskLSST/linearizer.20240926a,LSSTComCam/calib/DM-46360/isrTaskLSST/bfk.20240926a,LSSTComCam/calib/DM-46360/isrTaskLSST/ptc.20240926a
+export TAG=DP1
+export RERUN=20250207a
+export BOOTSTRAP_RUN_NUMBER=1
+
+# These are certification env vars
+export START_DATE=1970-01-01T00:00:00
+export CERT_RERUN=20250207a
+
+export SELECTION_BIAS="instrument='LSSTComCam' and ((exposure.day_obs=20241208 and exposure.seq_num >= 696) or (exposure.day_obs=20241206 and exposure.seq_num >=427)) and exposure.observation_type='bias'"
+export SELECTION_DARK="instrument='LSSTComCam' and ((exposure.day_obs=20241208 and exposure.seq_num >= 696) or (exposure.day_obs=20241206 and exposure.seq_num >=427)) and exposure.observation_type='dark'"
+export SELECTION_FLAT_BOOTSTRAP="instrument='LSSTComCam' and exposure.day_obs=20240806 and exposure.science_program='BLOCK-T68' and exposure.observation_reason='flat' and physical_filter='r_03'"
+
+export SELECTION_PTC_CTI="exposure.day_obs=20240806 and instrument='LSSTComCam' and exposure.science_program='BLOCK-T68' and exposure.observation_reason='comcam_ptc' and physical_filter='r_03'"
+
+export SELECTION_FLAT_u="instrument='$INSTRUMENT' and exposure.day_obs=20241110 and exposure.seq_num in (44..63)"
+export SELECTION_FLAT_g="instrument='$INSTRUMENT' and exposure.day_obs=20241126 and exposure.seq_num in (21..40)"
+export SELECTION_FLAT_r="instrument='$INSTRUMENT' and exposure.day_obs=20241111 and exposure.seq_num in (17..36) and band='r'"
+export SELECTION_FLAT_i="instrument='$INSTRUMENT' and exposure.day_obs=20241120 and exposure.seq_num in (41..64) and exposure.seq_num != 58"
+export SELECTION_FLAT_z="instrument='$INSTRUMENT' and exposure.day_obs=20241112 and exposure.seq_num in (28..48)"
+export SELECTION_FLAT_y="instrument='$INSTRUMENT' and exposure.day_obs=20241119 and exposure.seq_num in (21..36)"
+
+cd /sdf/data/rubin/user/erykoff/calibrations/LSSTComCam/DM-48520
+mkdir -p images
+
+cat $CP_PIPE_DIR/bps/templates/bps_biasBootstrap.yaml | envsubst
```

Comment:
Can this have a comment explaining that this is to visually check that all of the variables are defined for the BPS input?  I think having a comment on at least the first instance of each type of command call would be useful.